.gitmodules

```diff
@@ -13,3 +13,9 @@
 [submodule "contracts/lib/canonical-weth"]
 	path = contracts/lib/canonical-weth
 	url = https://github.com/Snowfork/canonical-weth
+[submodule "lodestar"]
+    path = lodestar
+    url = https://github.com/ChainSafe/lodestar
+[submodule "go-ethereum"]
+    path = go-ethereum
+    url = https://github.com/ethereum/go-ethereum
```

Comment:
The convention is that we use Nix where possible for local E2E dependencies. I believe we've discussed this before.

Nix has support for a very recent go-ethereum: https://search.nixos.org/packages?channel=unstable&show=go-ethereum&from=0&size=50&sort=relevance&type=packages&query=go-ethereum

Reply:
What I'm testing with in this PR is 1.13.5 https://github.com/Snowfork/snowbridge/blob/cce771451a53260523bc5da6efc46c60ebb51f10/relayer/go.mod#L7
but I think 1.13.4 as the recent nix package in the search result should also be fine.

Some other reason why I prefer to fork the upstream repos in https://github.com/Snowfork/snowbridge/pull/981#discussion_r1401401017